solutionbox/code_free_ml/mltoolbox/code_free_ml/analyze.py

```diff
@@ -158,6 +158,76 @@ def parse_arguments(argv):
   return args
 
 
+def make_analysis_plan(schema, features):
```

Comment:
Is the map as simple as the following?

if transform in CATEGORICAL_TRANSFORMS:
  ...
elif transform in NUMERIC_TRANSFORMS:
  ...

Then "plan" is not different from transforms group that you created in constants. When I looked at the code, it took me a while to figure out what "plan" is.